src/libData/AccountData/AccountStoreSC.h

```diff
@@ -64,6 +64,9 @@ class AccountStoreSC : public AccountStoreBase<MAP> {
   /// the blocknum while executing each txn
   uint64_t m_curBlockNum{0};
 
+  /// the DS blocknum
+  uint64_t m_curDSBlockNum{0};
```

Comment:
variable is not really set anywhere. Is it just a placeholder?

Reply:
indeed, the reason is that Zilliqa has more than one blocknumber - and EVM has just one NUMBER opcode ( https://www.ethervm.io/#43), because one Ethereum block type.

So added both block numbers, but only one is really used. Until we decide how to support both in the EVM. 

So really a placeholder for now.

Reply:
It is zero by default, so should be safe regardless.